zks-crypto-c/src/lib.rs

```diff
@@ -148,3 +149,25 @@ pub extern "C" fn zks_crypto_sign_musig(
     }
     MUSIG_SIGN_RES::MUSIG_SIGN_OK
 }
+
+#[repr(C)]
+pub enum MUSIG_VERIFY_RES {
+    MUSIG_VERIFY_OK = 0,
+    MUSIG_VERIFY_FAILED,
+}
```

Comment:
Any reason why we use this particular enum instead of boolean? Correct me if I'm wrong, usually, enums are used to represent success and multiple types of errors, whereas here we have just one type of error.

If it's to make a more uniform library interface (where _every_ function returns an enum) then I don't have any objections

Reply:
Actually I think so too (You can see correct solution in schnorr musig) But previous developer of this library decided implement it like this as separate structure for each  return code type. [Look at this commit](https://github.com/zksync-sdk/zksync-crypto-c/commit/0bde3ec410cec25539c33e3c441521cb06bf3ef0). Changing that was out of scope tasks.

Reply:
I guess it's ok either way, we just have to document it somewhere that 0 in this case means successful verification

Reply:
According [POSIX](https://www.gnu.org/software/libc/manual/html_node/Exit-Status.html) standard exit codes we can expect that success return code is always zero. But the better way is create supporting documentation for this library.

In additional I'll add short description of return codes into source code